lib/OpenQA/Schema.pm

```diff
@@ -75,7 +75,7 @@ sub dsn {
     return $self->storage->connect_info->[0]->{dsn};
 }
 
-sub deployment_check {
+sub deployment {
```

Comment:
This function does the actual deployment (and does not return an object related to the deployment). So I would rather call it `deploy`.

Reply:
yes, that makes sense

Reply:
done